### DISCUSSION TITLE
How to import data files

### DISCUSSION THREAD

**Discussion by tianxiezuo123** [ansys/pyfluent#1534] [Q&A]
I know how to import a case file，like: solver.file.read(file_type="case", file_name="XXX.cas")
but i dont know how to import data file 
I try to use : solver.file.read(file_type="dats", file_name="xxx.dat")
It reports an error : RuntimeError: Work in progress or unsupported file type Error Object: dats

**Comment by tianxiezuo123** [comment]
i solve this problem

**Reply by zouki-dev** [reply]
Can you share ?!
